### PR TITLE
Fixes #1135: Output of image links is not correct

### LINKED ISSUE #1135: Output of image links is not correct
- Issue type: Bug report

When we do api calls for '/api/search.json?q=query' , in image attribute (in json output), tweet post links also appear with image links.

Like for call loklak.org/api/search.json?q=fossasia , 

screenshot from 2017-05-08 02-16-07

### PR DESCRIPTION
**Pull request by vibhcool**
Fixes issue #1135 

### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: src/org/loklak/harvester/TwitterScraper.java [hunk 1]
```diff
@@ -332,26 +332,14 @@ private static String prepareSearchUrl(final String query, final ArrayList<Strin
...
-            // get images
-            if ((p = input.lastIndexOf("background-image:url('")) >= 0) {
-                int q = input.lastIndexOf("'");
-                if (q > p + 22) {
-                    String image_url = input.substring(p + 22, q);
-                    if (!image_url.endsWith(".jpg") && !image_url.endsWith(".png")) {
-                        DAO.log("strange image url: " + image_url);
-                    }
-                    images.add(image_url);
+            String img_link;
```
Comment (singhpratyush) [inline src/org/loklak/harvester/TwitterScraper.java]: The scope can be reduced here.

Also, please provide some sample results (screenshot/link).

**Comment by vibhcool** [comment]
screenshot from 2017-09-03 23-10-02

File: src/org/loklak/harvester/TwitterScraper.java (hunk 1 above)
Comment (vibhcool) [reply src/org/loklak/harvester/TwitterScraper.java]: uploaded the screenshot, please see

File: src/org/loklak/harvester/TwitterScraper.java [hunk 2]
```diff
@@ -332,26 +332,14 @@ private static String prepareSearchUrl(final String query, final ArrayList<Strin
...
+            if(videos == null) images = new HashSet<>();
+            if ((p = input.indexOf("<img")) >= 0) {
+                img_link = new prop(input, p, "src").value;
+                if (img_link != null && img_link.contains("pbs.twimg.com/media/")) {
```
Comment (Achint08) [inline src/org/loklak/harvester/TwitterScraper.java]: Will this not include even profile picture of the account?

Comment (vibhcool) [reply src/org/loklak/harvester/TwitterScraper.java]: nopes, only from tweet

File: src/org/loklak/harvester/TwitterScraper.java (hunk 1 above)
Comment (vibhcool) [reply src/org/loklak/harvester/TwitterScraper.java]: @singhpratyush , I have fixed this issue, please see :)

**Pull request merged**